modules/ocf_desktop/manifests/firewall_output.pp

```diff
@@ -0,0 +1,24 @@
+#allow desktops to send packets to papercut, pagefault, radiation
```

Comment:
why radiation?

Reply:
That's a good question, because I don't believe they ever have to access radiation directly, only laptops or phones should need access to it.